### PR TITLE
Dockerization Refactoring

### LINKED ISSUE #49: Dockerization Refactoring

### PR DESCRIPTION
**Pull request by use-to**
closes #49

### CODE REVIEW

**Comment by use-to** [comment]
related to #50

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+FROM alpine AS production
+
+LABEL maintainer="development@helpwave.de"
+
+WORKDIR /app
+
+COPY --from=builder /app/helpwave .
+RUN chmod +x helpwave
+
+RUN touch .env
```
Comment (FoseFx) [inline Dockerfile:18]: Is this needed? If so we should probably fix that

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+
+ENV PORT 80
+ENV GIN_MODE release
+ENV LOG_LEVEL info
+ENV POSTGRES_PORT 5432
```
Comment (FoseFx) [inline Dockerfile:23]: Why set POSTGRES_PORT in the container, but everything else in the docker-compose? I don't think setup specific variables should be set in the container.

**Review by FoseFx — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (use-to) [reply Dockerfile:18]: Yes, it is. We should not depend on `.env` files at all.

File: Dockerfile (hunk 2 above)
Comment (use-to) [reply Dockerfile:23]: Becuase `5432` is the default port set for postgresql. Keep in mind that you can still overwrite this later on using the `environment`-option in `docker-compose` (for example).

File: Dockerfile (hunk 1 above)
Comment (use-to) [reply Dockerfile:18]: Created an issue for that: #52

**Pull request merged**